Jenkinsfile

```diff
@@ -0,0 +1,303 @@
+pipeline {
+    parameters {
+        booleanParam defaultValue: false,
+                description: 'Whether to upload the packages in playground repositories',
+                name: 'PLAYGROUND'
+    }
+    options {
+        skipDefaultCheckout()
+        buildDiscarder(logRotator(numToKeepStr: '5'))
+        timeout(time: 3, unit: 'HOURS')
+    }
+    agent {
+        node {
+            label 'base-agent-v1'
+        }
+    }
+    environment {
+        NETWORK_OPTS = '--network ci_agent'
+        ARTIFACTORY_ACCESS=credentials('artifactory-jenkins-gradle-properties-splitted')
+    }
+    stages {
+        stage('Checkout') {
+            steps {
+                checkout scm
+            }
+        }
+        stage('Stash') {
+            steps {
+                stash includes: 'package/**, conf/**', name: 'staging'
+            }
+        }
+        stage('Package') {
+            parallel {
+                stage('Ubuntu 20.04') {
+                    agent {
+                        node {
+                            label 'yap-agent-ubuntu-20.04-v2'
+                        }
+                    }
+                    steps {
+                        unstash 'staging'
+                        sh 'cp -r staging /tmp'
+                        sh 'sudo yap build ubuntu-focal /tmp/staging/package'
+                        stash includes: 'artifacts/',
+                                name: 'artifacts-ubuntu-focal'
+                    }
+                    post {
+                        always {
+                            archiveArtifacts artifacts: 'artifacts/*focal*.deb',
+                                    fingerprint: true
+                        }
+                    }
+                }
+                stage('Ubuntu 22.04') {
+                    agent {
+                        node {
+                            label 'yap-agent-ubuntu-22.04-v2'
+                        }
+                    }
+                    steps {
+                        unstash 'staging'
+                        sh 'cp -r staging /tmp'
```

Comment:
cp -r staging /tmp is not needed.
next line become

sudo yap build ubuntu-focal package

Reply:
Thanks, I know it can be simplified, it is in this way only to make build_packages.sh work to test package changes locally without need to push on CI!

Reply:
Applied suggested changes as in carbonio-core repo, thanks!